emails/management/commands/get_delayed_emails_from_sqs.py

```diff
@@ -0,0 +1,45 @@
+import json
+import logging
+import sys
+
+from botocore.exceptions import ClientError
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+
+from emails.views import (
+    _sns_inbound_logic, validate_sns_header, verify_from_sns
+)
+
+
+logger = logging.getLogger('events')
+
+class Command(BaseCommand):
+    help = 'Fetches messages from SQS dead-letter queue and processes them.'
+
+    def handle(self, *args, **options):
+        self.exit_code = 0
+        try:
+            dl_queue = apps.get_app_config('emails').dl_queue
+        except ClientError as e:
+            logger.error('sqs_client_error: ', extra=e.response['Error'])
+            self.exit_code = 1
+            sys.exit(self.exit_code)
+
+        messages = dl_queue.receive_messages(
+            MaxNumberOfMessages=10, WaitTimeSeconds=1
+        )
+        while len(messages) > 0:
+            for message in messages:
+                json_body = json.loads(message.body)
+                verified_json_body = verify_from_sns(json_body)
+                topic_arn = verified_json_body['TopicArn']
+                message_type = verified_json_body['Type']
+                validate_sns_header(topic_arn, message_type)
```

Comment:
@groovecoder Do we need this since it's not coming from SNS? I would assume SQS payloads fail this check, but there's no HTTP response to give?

Reply:
Oh actually looking at an example payload it keeps all of the SNS data so this is probably good validation. Disregard :slightly_smiling_face:

Reply:
Yeah, the SNS-receiving code checks these via HTTP request headers that SNS sends. I didn't want to remove these checks completely, and the same values are also sent in the JSON body, so I just grabbed them out of the JSON body and run the same validation code on them.